rhizome/host/lib/cloud_hypervisor.rb

```diff
@@ -13,10 +14,38 @@ def path
     end
   }
 
+  FirmwareClass = Struct.new(:arch, :version) {
```

Comment:
Should we pass `arch` here? It's rhizome code and only has a single architecture.